internal/docs/yaml.go

```diff
@@ -18,6 +18,7 @@ type Command struct {
 	Usage       string
 	Aliases     []string
 	Examples    string
+	Slug        string
```

Comment:
Some of these things are shared between `yaml` and `mdx`, what do you think about creating a `docs.go` file for the shared structs & functions?